src/scripts/oauth_tables_script.sql

```diff
@@ -0,0 +1,110 @@
+
+DROP TABLE IF EXISTS app_clients CASCADE;
+
+CREATE TABLE app_clients(
+             id BIGSERIAL NOT NULL,
+             client_name VARCHAR(50) UNIQUE NOT NULL,
+             client_id VARCHAR(50) UNIQUE NOT NULL,
+             client_secret VARCHAR(50) UNIQUE NOT NULL,
```

Comment:
50 is not a good idea for a secret, this can be a really long text

Reply:
done.